scripts.js

```diff
@@ -15,4 +15,251 @@ const navLinks = document.querySelectorAll('.nav-item');
+  const langList = document.createElement('ul');
+  languages.appendChild(langList);
+
+  for (let i = 0; i < card.list2.length; i += 1) {
+    const list2Items = document.createElement('li');
+    list2Items.className = 'lang--names';
+    list2Items.textContent = card.list2[i];
+    langList.appendChild(list2Items);
+  }
+  const cardBtn = document.createElement('div');
+  cardBtn.className = 'works';
+  cardContent.append(cardBtn);
+  const btnLink = document.createElement('a');
+  btnLink.className = 'displayModal';
+  cardBtn.appendChild(btnLink);
+  const btnTxt = document.createElement('p');
+  btnTxt.textContent = 'See Project';
+  btnLink.appendChild(btnTxt);
+});
+
+cards.forEach((card) => {
+  const [,,, li4] = card.list1;
+  const modal = document.createElement('div');
+  modal.className = 'modal';
+  document.querySelector('#portfolio').appendChild(modal);
+  const content = document.createElement('div');
+  content.className = 'modal-content';
+  modal.appendChild(content);
+  const closebtn = document.createElement('span');
+  closebtn.innerHTML = '&times;';
+  content.appendChild(closebtn);
+  closebtn.className = 'closeModal';
+  const heading2 = document.createElement('h2');
+  content.appendChild(heading2);
+  heading2.className = 'modalh2';
+  const canopyDiv = document.createElement('div');
+  canopyDiv.className = 'canopy2 canopymodal';
+  content.appendChild(canopyDiv);
+  const mobileImage = document.createElement('img');
+  mobileImage.className = 'mobile_image enlm';
+  mobileImage.src = card.deskImage;
+  content.appendChild(mobileImage);
+  const DeskImage = document.createElement('img');
+  DeskImage.className = 'desktop_image  enl';
+  DeskImage.src = card.deskImage;
+  content.appendChild(DeskImage);
+  const ContentDiv = document.createElement('div');
+  ContentDiv.className = 'flex_content pop';
+  content.appendChild(ContentDiv);
+  const heading3 = document.createElement('h3');
+  canopyDiv.appendChild(heading3);
+  heading3.className = 'modalHeader';
+  heading3.textContent = card.heading3;
+  const paragraph = document.createElement('p');
+  paragraph.className = 'card-content pop_paragh';
+  ContentDiv.appendChild(paragraph);
+  paragraph.textContent = card.paragraph;
+  const list1 = document.createElement('ul');
+  list1.className = 'class-content ul1_modal ';
+  canopyDiv.appendChild(list1);
+  const classes = ['canopy', 'back-end', 'year'];
+  for (let j = 0, i = 0; j < card.list1.length - 1; j += 1, i += 1) {
+    const list1Elements = document.createElement('li');
+    list1Elements.className = classes[i];
+    list1Elements.textContent = card.list1[j];
+    list1.appendChild(list1Elements);
+
+    if (j < card.list1.length - 2) {
+      const list1items = document.createElement('l1');
+      list1items.className = 'dot';
+      const dotz = document.createElement('img');
+      dotz.src = li4;
+      list1items.appendChild(dotz);
+      list1.appendChild(list1items);
+    }
+  }
```

Comment:
:red_circle: [Required] 
Please Make sure when rendering the pop up you ship the right content for each item. As your pop-up keeps the same data every time it pops in. You should update the state of each part depending on the source item's button. The goal of this is to associate each project with the popup window details
This is actually what is happening :arrow_down: 

![Peek 2022-11-26 14-09](https://user-images.githubusercontent.com/26736582/204088378-1f0fe982-7221-4aa8-98fc-01f7ca8e7614.gif)